spec/rubocop/cop/rspec/capybara/specific_matcher_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::Capybara::SpecificMatcher do
+  it 'does not register an offense for abstract matcher when elements with' \
+    'no matcher specified in detail' do
+    expect_no_offenses(<<-RUBY)
+      expect(page).to have_selector('article')
+      expect(page).to have_none_of_selectors('body')
+      expect(page).to have_css('tbody')
+    RUBY
+  end
+
+  it 'does not register an offense for abstract matcher when elements with' \
```

Comment:
`\` combines this into a string without adding a space:
> does not register an offense for abstract matcher when elements withnot element

Reply:
Oops, I'll add that space.